tests/multigrid-transfer/test.h

```diff
@@ -0,0 +1,167 @@
+// ---------------------------------------------------------------------
```

Comment:
I just realized that this is a new folder. Don't you need to add a CMakeList.txt? Also, maybe we should name this folder multigrid-global-coarsening ?